repos/fdbt-site/src/pages/managePurchaseMethod.tsx

```diff
@@ -21,44 +21,57 @@ const editingInformationText =
     'Editing and saving new changes will be applied to all fares using this purchase method.';
 
 // DistributionChannelType
-export const purchaseLocationsList = {
-    id: 'checkbox-0-on-board',
-    method: ['onBoard', 'online', 'mobileDevice', 'postal', 'agency'],
+
+export const purchaseLocationsList = (isCapped: boolean): { id: string; method: string[] } => {
+    return {
+        id: 'checkbox-0-on-board',
+        method: isCapped ? ['onBoard', 'mobileDevice'] : ['onBoard', 'online', 'mobileDevice', 'postal', 'agency'],
+    };
 };
 
-// Payment Method
-export const paymentMethodsList = {
-    id: 'checkbox-0-cash',
-    paymentMethods: [
-        'cash',
-        'debitCard',
-        'creditCard',
-        'mobilePhone',
-        'cheque',
-        'directDebit',
-        'contactlessTravelCard',
-    ],
+export const paymentMethodsList = (isCapped: boolean): { id: string; paymentMethods: string[] } => {
```

Comment:
Can we changed the isCapped boolean to be optional? so that we dont have to pass 'false' to it.

Reply:
For all 3 methods.